src/redfish_collector/core/rawCollector.py

```
@@ -202,83 +232,100 @@ async def dataCollector(serverAddress,username,password,templateDir,logLevel):
+            logging.info("[%s] We will generate data model with schema file: %s" % (serverAddress,modelSchema))
+        else:
+            logging.error("[%s] We couldn't find any schema similar with server model: %s. Please check schema directory" % (serverAddress,model))
+            return
+
+        # logging.info(vendorData)
+        modelSchemaDir = templateDir + "schemas/" + modelSchema
+        schema=readYAMLTemplate(modelSchemaDir, serverAddress)
+        # logging.info(schema)
+        dataNewSchema = schema['Data']
+        if schema is None:
+            logging.error("[%s] Can't generate vendor schema, please check again" % serverAddress)
```

Ruff (0.15.18)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)